.github/workflows/haskell.yml

```diff
@@ -137,9 +137,6 @@ jobs:
     - name: Build
       run: retry 2 cabal build cardano-node cardano-cli cardano-node-chairman cardano-submit-api plutus-example
 
-    - name: Git clone
```

Comment:
This should result in a significant speed up right?

Reply:
Yes, but also CI currently fails because the repository is many gigabytes large and the CI VM runs out of disk space.